src/poly1305.c

```diff
@@ -9,9 +9,7 @@
 /* $OpenBSD: poly1305.c,v 1.3 2013/12/19 22:57:13 djm Exp $ */
 
 #include <sys/types.h>
-#ifdef HAVE_STDINT_H
 # include <stdint.h>
```

Comment:
`stdint.h` is missing from earlier MSVC versions.
But `libssh2.h` already has the necessary compatilbility tricks added,
thus including it may fix this. It also includes `sys/types.h` already.

Reply:
Ah, you fixed already!